### PR TITLE
BCTS Local DAP Development Environment

### PR DESCRIPTION
**Pull request by smunthik**
- Please refer the readme.md
- Local development environment for DAP for BCTS data works

### CODE REVIEW

File: init-scripts/init_postgres_schema.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+\connect ods
+
+-- Create schema
+CREATE SCHEMA IF NOT EXISTS FOREST;
+
+-- Create user
+CREATE USER ods_proxy_user_lrm WITH PASSWORD 'ods_proxy_user_lrm';
+
+-- Grant privileges
+GRANT USAGE ON SCHEMA FOREST TO ods_proxy_user_lrm;
```
Comment (abimichel) [inline init-scripts/init_postgres_schema.sql:11]: Great set up for testing! Schemas in ODS follow naming convention <app>_replication so you will likely be dealing with fta_replication and lrm_replication :)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,124 @@
...
+
+    Fork the repository.
+    Create a feature branch.
+    Make your changes.
+    Submit a pull request.
+
+### License
+This project is licensed under the MIT License.
+
+### Acknowledgments
```
Comment (abimichel) [inline README.md:117]: Love it

File: README.md [hunk 3]
```diff
@@ -1,2 +1,124 @@
...
+
+2. **Generate DDL**:
+    Initialize the local Oracle database with the required source tables by running the python script, init-scripts\init_oracle_import.py
+    Generate DDL scripts from local FOREST schema. Convert to PostgreSQL syntax.
+    Use DDL scripts generated to create the required target tables in the local ODS.
+
+3. **Trigger DAGs**:
+
+    Ensure K8s cluster is running on Docker desktop
+    Trigger the DAGs to replicate the data from Oracle to ODS.
```
Comment (abimichel) [inline README.md:64]: For this step, we have an existing container that works with the Kubernetes pod operator. To grab it: 

docker pull ghcr.io/bcgov/nr-dap-ods-ora2pg:main

I'll ping you with the env vars / secrets to test with.

File: dags/test_dag.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+default_args = {
+    'owner': 'airflow',
+    'start_date': datetime(2023, 1, 1),
+}
+
+with DAG('test_k8s_dag', default_args=default_args, schedule_interval=None) as dag:
+
+    k8s_task = KubernetesPodOperator(
+        namespace='default',
+        image='python:3.8-slim',
```
Comment (abimichel) [inline dags/test_dag.py:14]: Feel free to use any Python image for your transformations, etc 

As mentioned in previous comments, the image 'ghcr.io/bcgov/nr-dap-ods-ora2pg:main' is available to use for basic, like-to-like replication too.

**Review by abimichel — APPROVED** [review]

**Review by Krishna-R-BC — APPROVED** [review]

File: init-scripts/init_postgres_schema.sql (hunk 1 above)
Comment (smunthik) [reply init-scripts/init_postgres_schema.sql]: Thanks @abimichel! I updated the schema name to `lrm_replication`

**Pull request merged**